### PR TITLE
Add support for uint and gpb.TypedValue in protomap

### PR DESCRIPTION
**Pull request by robshakir**
```
commit 727e911106addf67bd05c7d60e8a0e168628712b
Author: Rob Shakir <robjs@google.com>
Date:   Tue Jun 8 02:06:09 2021 -0700

    Add support for mapping typedvalues, and uint fields.
    
     * (M) protomap/proto.go
     * (M) protomap/proto_test.go
       - Accept an input TypedValue such that we can parse gNMI
         Notifications directly.
       - Add handling for Uint values.
```

### CODE REVIEW

File: protomap/proto.go [hunk 1]
```diff
@@ -430,6 +449,20 @@ func makeWrapper(msg protoreflect.Message, fd protoreflect.FieldDescriptor, val
 			return nil, false, fmt.Errorf("got non-string value for string field, field: %s, value: %v", fd.FullName(), val)
 		}
 		return (&wpb.StringValue{Value: nsv}).ProtoReflect(), true, nil
+	case *wpb.UintValue:
+		var nsv uint64
+		switch {
+		case wasTypedVal:
+			nsv = val.(uint64)
+		default:
+			iv, ok := val.(int)
```
Comment (wenovus) [inline protomap/proto.go:458]: typo: uint? the test should be fixed as well.

File: protomap/proto.go [hunk 2]
```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/openconfig/ygot/ygot"
 
 	gpb "github.com/openconfig/gnmi/proto/gnmi"
+	"github.com/openconfig/gnmi/value"
```
[the remark this answers is not in the document]
Comment (robshakir) [reply protomap/proto.go:32]: Done, thanks.

File: protomap/proto.go (hunk 1 above)
Comment (robshakir) [reply protomap/proto.go:458]: Yes, thanks!

**Comment by wenovus** [comment]
Thanks for addressing the comments, once you push your changes I'll approve.

**Comment by robshakir** [comment]
Thanks - everything should be pushed here. Apologies, not sure how I missed that.

**Review by wenovus — APPROVED** [review]

**Pull request merged**